packages/website/src/templates/doc-page.js

```diff
@@ -35,9 +48,18 @@ function DocPageTemplate({ data, location, ...props }) {
     <Layout pathname={location.pathname}>
       <MDXProvider components={components}>
         <RC.Box>
-          <RC.Heading size={3} fontWeight="bold">
+          <RC.Heading size={4} fontWeight="bold" mb={2} lineHeight={3}>
             {data.mdx.frontmatter.title}
           </RC.Heading>
+          <RC.Heading
+            size={1}
+            mb={7}
+            as="h2"
+            lineHeight={2}
+            style={{ color: 'hsl(209, 14%, 46%)' }}
+          >
+            This is the description pulled from the primitive. It can be edited here by the user.
```

Comment:
This is currently showing on all pages. A bit weird 🤔 

Want me to move it into the `mdx` files? Could be a metadata on them, or could just be a `##` (`h2`)

Reply:
Sure. Eventually, it should be pulled from the primitive file by default, then editable in the GUI.